examples/point-of-sale/package.json

```diff
@@ -20,47 +20,46 @@
     },
     "dependencies": {
         "@solana/pay": "^0.2.4",
-        "@solana/spl-token": "^0.3.5",
-        "@solana/wallet-adapter-base": "^0.9.16",
-        "@solana/wallet-adapter-react": "^0.15.18",
-        "@solana/wallet-adapter-react-ui": "^0.9.16",
-        "@solana/wallet-adapter-wallets": "^0.18.9",
-        "@solana/web3.js": "^1.56.0",
-        "bignumber.js": "^9.1.0",
-        "bs58": "^4.0.1",
-        "cache-manager": "^4.1.0",
+        "@solana/spl-token": "^0.3.6",
+        "@solana/wallet-adapter-base": "^0.9.19",
+        "@solana/wallet-adapter-react": "^0.15.25",
+        "@solana/wallet-adapter-react-ui": "^0.9.23",
+        "@solana/wallet-adapter-wallets": "^0.19.7",
+        "@solana/web3.js": "^1.69.0",
+        "bignumber.js": "^9.1.1",
+        "bs58": "^5.0.0",
+        "cache-manager": "^5.1.4",
         "clsx": "^1.2.1",
         "color-interpolate": "^1.0.5",
         "cors": "^2.8.5",
-        "date-fns": "^2.29.2",
-        "express-rate-limit": "^6.5.2",
+        "date-fns": "^2.29.3",
+        "express-rate-limit": "^6.7.0",
         "react": "^18.2.0",
         "react-circular-progressbar": "^2.1.0",
         "react-dom": "^18.2.0",
-        "react-responsive": "^9.0.0-beta.10"
+        "react-responsive": "^9.0.2"
     },
     "devDependencies": {
         "@types/bs58": "^4.0.1",
-        "@types/cache-manager": "^4.0.1",
-        "@types/cors": "^2.8.12",
+        "@types/cache-manager": "^4.0.2",
+        "@types/cors": "^2.8.13",
         "@types/eslint-plugin-prettier": "^3.1.0",
-        "@types/express": "^4.17.13",
-        "@types/express-rate-limit": "^6.0.0",
-        "@types/node": "^18.7.14",
-        "@types/prettier": "^2.7.0",
-        "@types/react": "^18.0.18",
-        "@types/react-dom": "^18.0.6",
-        "eslint": "^8.23.0",
-        "eslint-config-next": "^12.2.5",
+        "@types/express": "^4.17.14",
+        "@types/node": "^18.11.11",
+        "@types/prettier": "^2.7.1",
+        "@types/react": "^18.0.26",
+        "@types/react-dom": "^18.0.9",
+        "eslint": "^8.29.0",
+        "eslint-config-next": "^13.0.6",
         "eslint-config-prettier": "^8.5.0",
         "eslint-plugin-prettier": "^4.2.1",
-        "local-ssl-proxy": "^1.3.0",
-        "next": "^12.2.5",
+        "local-ssl-proxy": "^1.2.3",
```

Comment:
Why was this downgraded?

Reply:
Can't remember why, but agree on this one too!
Built and working great with last version.